minimax/README.md

```diff
@@ -22,3 +22,13 @@ $ ginkgo bootstrap
 $ ginkgo generate minimax
 ```
 
+## Thoughts
+
+- 5 test cases.  _Am I done already???_
```

Comment:
In other words, I was pleasantly surprised that it only took 5 test cases to get to the final algorithm.